spec/acceptance/tictactoe_spec.rb

```diff
@@ -159,7 +142,27 @@ class InMemoryPlayerGateway
     expect(check_board.execute(board)). to eq('X has won')
   end
 
-  it 'determines the winner when theres a O match across' do
+  it 'determines the winner when theres a X match diagonal backward' do
+    place_mark.execute(player: :X, x: 0, y: 2)
+    view_board.execute({})
+
+    place_mark.execute(player: :O, x: 0, y: 1)
+    view_board.execute({})
+
+    place_mark.execute(player: :X, x: 1, y: 1)
+    view_board.execute({})
+
+    place_mark.execute(player: :O, x: 1, y: 2)
+    view_board.execute({})
+
+    place_mark.execute(player: :X, x: 2, y: 0)
+    response = view_board.execute({})
+    board = response[:board]
+
+    expect(check_board.execute(board)). to eq('X has won')
```

Comment:
Id expect `check_board.execute` to return a hash, not a string so:

`expect(check_board.execute(board)). to eq({ status: 'X has won' })`

Also, I'd avoid using strings (they imply a message that will be displayed to the user).

Instead, use symbols e.g.

`expect(check_board.execute(board)). to eq({ status: :x_wins })`